lib/library.rb

```diff
@@ -0,0 +1,37 @@
+require 'date'
+
+class Library
+  STANDARD_VALIDITY_MONTHS = 1
+
+  attr_accessor :collection, :exp_date, :shelf
+
+  def initialize
+    @collection = YAML.load_file('./books/data.yml')
+    @exp_date = Date.today.next_month(STANDARD_VALIDITY_MONTHS).strftime('%d/%m')
+  end
+
+  def checkout
+    if collection.select { |obj| obj[:available]}
+      'Book is available for checkout'
+
+    else
+      'Book is not available for checkout'
+    end
+  end
```

Comment:
Just returning a statement saying whatever a book is available or not, is not relevant to this method if we look at the method name that indicates that a checkout should be performed, right?